tests/km_core_tests.bats

```diff
@@ -584,9 +584,9 @@ fi
       --ex="target remote :$km_gdb_port" \
       --ex="source cmd_for_protected_mem_test.gdb" --ex=q
    assert_success
-   assert_line --partial "first word  0x7fffffbfc000:	0x1111111111111111"
-   assert_line --partial "spanning pages  0x7fffffbfcffc:	0xff0000ffff0000ff"
-   assert_line --partial "last word  0x7fffffbfdff8:	0xeeeeeeeeeeeeeeee"
+   assert_line --regexp "first word  0x7ffff?f?bfc000:	0x1111111111111111"
+   assert_line --regexp "spanning pages  0x7ffff?f?bfcffc:	0xff0000ffff0000ff"
+   assert_line --regexp "last word  0x7ffff?f?bfdff8:	0xeeeeeeeeeeeeeeee"
```

Comment:
Could you add a note to explain why the "?" in the assert_line's are there.
Whoever looks at this in a year is probably wonder why this is here.